src/main/java/com/commercetools/service/paypalPlus/impl/BasePaypalPlusService.java

```diff
@@ -51,15 +53,16 @@ protected Logger getLogger() {
     protected final <R> CompletionStage<R> paypalPlusStageWrapper(
             @Nonnull PayPalRESTExceptionSupplier<APIContext, R> supplier)
             throws PaypalPlusServiceException {
+        ExtendedAPIContext extendedAPIContext = paypalPlusExtendedApiContextFactory.createAPIContext();
+        String tenantName = extendedAPIContext.getTenantName();
+
         return CompletableFuture.supplyAsync(() -> {
             try {
-                APIContext context = paypalPlusApiContextFactory.createAPIContext();
-                return supplier.apply(context);
+                return supplier.apply(extendedAPIContext.getApiContext());
             } catch (PayPalRESTException e) {
-                throw new PaypalPlusServiceException("Paypal Plus REST service exception", e);
+                throw new PaypalPlusServiceException("Paypal Plus REST service exception. Tenant: " + tenantName, e);
```

Comment:
There's no test for what you did - that the error message will include the tenant name.

Reply:
d2716c3a3d822e3808a89c7e51041496c1cde9b2